src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -518,7 +518,7 @@ function GlobeControls(view, target, radius, options = {}) {
             targetDistance *= Math.tan((this.camera.fov / 2) * Math.PI / 180.0);
 
             // we actually don't use screenWidth, since perspective camera is fixed to screen height
-            this.panLeft(2 * deltaX * targetDistance / element.clientHeight);
+            this.panLeft(2 * deltaX * targetDistance / element.clientWidth);
```

Comment:
actually I'm not sure about that. `targetDistance` is calculated from the camera fov, which is the *vertical* opening of the camera. Hence we normalize with `element.clientHeight`. 

For this reason (I believe), according to my tests, the mouse follow the ground better with the old code when you do some left-right pan. Thoughts?

Reply:
To precise things after our irc discussion, what I've said above is only true because `camera.aspect` is correctly set to `width / height`. I guess this won't work if someone wants to play with a different aspect (have a skewed scene). The correct calculus should then be:

```js
this.panLeft(2 * deltaX * targetDistance * camera.aspect / element.clientWidth);
```

(sanity check, if camera.aspect is `element.clientWidth / element.clientHeight`, then it's the same as `2 * deltaX * targetDistance / element.clientHeight`).

Do you want to amend your PR with this calculus?

Reply:
OK, I changed the file as you suggested.
Note that it doesn't change anything in our case right now, but it will work correctly if one allows to change the ratio `width / height`